cinema/serializers.py

```diff
@@ -53,12 +51,11 @@ class Meta:
 
 class MovieSessionListSerializer(MovieSessionSerializer):
     movie_title = serializers.CharField(source="movie.title", read_only=True)
-    cinema_hall_name = serializers.CharField(
-        source="cinema_hall.name", read_only=True
-    )
+    cinema_hall_name = serializers.CharField(source="cinema_hall.name", read_only=True)
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
+    tickets_available = serializers.IntegerField()
```

Comment:
The `tickets_available` field is defined but not sourced from any model attribute or method. Ensure that this field is calculated or retrieved correctly, possibly by overriding the `get_tickets_available` method in the serializer.